src/App.tsx

```diff
@@ -1,61 +1,125 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, User } from './types';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+
+const findNewTodoId = (todoArr: Todo[]) => {
+  todoArr.sort((todo1, todo2) => todo2.id - todo1.id);
+
+  return todoArr[0].id + 1;
+};
```

Comment:
The `findNewTodoId` function sorts the entire `todoArr` array to find the highest ID. This could be inefficient for large arrays. Consider using `Math.max` with `map` to find the maximum ID more efficiently.